index.js

```
@@ -124,11 +125,14 @@ export const validate = (obj) => {
  * @returns {Buffer} protobuf encoded buffer with dataTypeIdSize + schemaVersionSize bytes prepended, one for the type of record and the other for the version of the schema */
 export const encode = (obj) => {
   const blockPrefix = encodeBlockPrefix({
-    dataTypeId: schemasPrefix[formatSchemaType(obj.type)],
-    schemaVersion: obj.schemaVersion === undefined ? 0 : obj.schemaVersion,
+    dataTypeId: schemasPrefix[formatSchemaType(obj.type)].dataTypeId,
+    // TODO: how to handle wrong/missing schemaVersions?
```

I think we could throw an error here if a schema type or type with a specific version isn't found.